cc/defs.bzl

```diff
@@ -94,6 +99,8 @@ def swift_cc_tool_library(**kwargs):
     copts = local_includes + copts
     kwargs["copts"] = copts + (kwargs["copts"] if "copts" in kwargs else [])
 
+    kwargs["features"] = _default_features() + (kwargs["features"] if "features" in kwargs else [])
```

Comment:
Does Starlark support `dict.get`?

https://docs.python.org/3/library/stdtypes.html#dict.get

If it I think this would become:

```suggestion
    kwargs["features"] = _default_features() + kwargs.get("features", [])
```

Reply:
It does indeed: https://bazel.build/rules/lib/dict#get.

Thanks this is much better.